docs/dev/libvirt-howto.md

```diff
@@ -163,7 +163,7 @@ This step is optional, but useful for being able to resolve cluster-internal hos
     ```sh
     echo server=/tt.testing/192.168.126.1 | sudo tee /etc/NetworkManager/dnsmasq.d/tectonic.conf
     ```
-3. `systemctl restart NetworkManager`
+3. `sudo systemctl restart NetworkManager`
```

Comment:
This isn't strictly necessary. Users should know how to escalate privileges if needed.

Reply:
Yes, but `sudo` isn't the only way to escalate privileges. Most major distributions include polkit today.

Reply:
On my RHEL 7.5 CSB, running this without `sudo` pops up an "Authentication Required" window.  That seems like an acceptable UX for me, and folks can use `sudo` (or other approaches) instead if they see fit.